cardshifter-server/src/main/java/com/cardshifter/server/model/MainServer.java

```diff
@@ -72,9 +85,9 @@ public Server start() {
         mods.loadExternal(mods.getDefaultModLocation());
         mods.loadExternal(Paths.get(config.getModsDirectory()));
 
-        if(mods.getAvailableMods().isEmpty()) {
+        if (mods.getAvailableMods().isEmpty()) {
         	logger.warn("No mods were loaded from the different folders. Please check your installation.");
-        	throw new IllegalStateException("No mod loaded");
+        	throw new NoModsLoadedException("No mod loaded");
```

Comment:
I think you could just use a default constructor for this exception since the message is just its name really.

```java
public NoModsLoadedException() {
    super("No mods loaded");
}

// and then
throw new NoModsException();
```

Sorry for being really nitpicky :/